package.json

```diff
@@ -14,5 +14,6 @@
     "@typescript-eslint/parser": "^4.28.2",
     "eslint": "^7.30.0",
     "eslint-config-airbnb-typescript": "^12.3.1"
-  }
+  },
+  "license": "GPLV3"
```

Comment:
Oh didn't mean it literally, create a LICENSE file more info here:

https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/licensing-a-repository